apple-pie/index.html

```diff
@@ -4,18 +4,67 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
+    <link rel="stylesheet" href="/apple-pie/styles/style.css">
     <!-- don't forget to link your styles -->
   </head>
   <body>
-    Apple Pies This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
-    be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! Ingredients 1 recipe pastry for a 9 inch double crust pie 1/2 cup unsalted butter 3 tablespoons
-    all-purpose flour 1/4 cup water 3 tablespoons all-purpose flour 1/4 cup water 1/2 cup white sugar 1/2 cup packed
-    brown sugar 8 Granny Smith apples - peeled, cored and sliced Directions Preheat oven to 425 degrees F (220 degrees
-    C). Melt the butter in a saucepan. Stir in flour to form a paste. Add water, white sugar and brown sugar, and bring
-    to a boil. Reduce temperature and let simmer. Place the bottom crust in your pan. Fill with apples, mounded
+  <header class="header"> 
+    <h1> <!-- <img id="apple-pie" src="images/apple-pie.jpg" /> --> Apple Pies </h1> 
+    </header>
+    <section>
+    <div>
+  <p>This was my grandmother's apple pie recipe. I have never seen another one <br> quite like it. It will always
+      be my favorite and has won me several first place <br> prizes in local competitions. I hope it becomes one of your
+      favorites as well! </p> 
+    </div>
+    <div>
+      <img class="image-recipe" src="images/recipe-info.png" />
+    </div>
+
+    <hr>
+  </section>
+    
+    <section class="Ingredients-sec">
+      <h3>Ingredients </h3>
+   <ul>
+  <li>- 1 recipe pastry for a 9 inch double crust pie </li>
+  <li>- 1/2 cup unsalted butter</li>   
+  <li>- 3 tablespoons all-purpose flour</li>    
+  <li>- 1/4 cup water </li>    
+  <li>- 3 tablespoons all-purpose flour</li>
+  <li>- 1/4 cup water </li>
+  <li>- 1/2 cup white sugar</li> 
+  <li>- 1/2 cup packed brown sugar</li> 
+  <li>- 8 Granny Smith apples - peeled, cored and sliced </li>
+</section>
```

Comment:
Here you opened a `<section>` tag and then a `<ul>` tag, but after when you're closing the tags, you closed them in the wrong order `</ul>` and `</section>` so instead of:

```
    <section class="Ingredients-sec">
      <h3>Ingredients</h3>
      <ul>
        <li>- 1 recipe pastry for a 9 inch double crust pie</li>
        ( ... )
      </section>
    </ul>
```

Do:
```
    <section class="Ingredients-sec">
      <h3>Ingredients</h3>
      <ul>
        <li>- 1 recipe pastry for a 9 inch double crust pie</li>
        ( ... )
      </ul>
    </section>
```